pkg/core/stateroot/module.go

```diff
@@ -184,6 +188,43 @@ func (s *Module) JumpToState(sr *state.MPTRoot) error {
 	return nil
 }
 
+// GC performs garbage collection.
+func (s *Module) GC(index uint32, store storage.Store) time.Duration {
+	if !s.mode.GC() {
+		panic("stateroot: GC invoked, but not enabled")
+	}
+	var removed int
+	var stored int64
+	s.log.Info("starting MPT garbage collection", zap.Uint32("index", index))
+	start := time.Now()
+	b := store.Batch()
```

Comment:
So now it's always one batch, but GC period doesn't have an upper bound. What if the user sets GCP to 100500K? The batch will be too large, so maybe it's better to limit the batch size or to limit GCP?

Reply:
Well, it's a configuration issue then.